test/integration/identity/validation/validatePublicKeysFactory.spec.js

```diff
@@ -162,33 +162,65 @@ describe('validatePublicKeysFactory', () => {
       const [error] = result.getErrors();
 
       expect(error.dataPath).to.equal('.data');
-      expect(error.keyword).to.equal('minLength');
+      expect(error.keyword).to.equal('pattern');
     });
 
-    it('should be no longer than 2048 character', () => {
-      publicKeys[1].data = Buffer.alloc(2000).toString('base64');
+    describe('ECDSA_SECP256K1', () => {
+      it('should be no less than 44 character', () => {
+        publicKeys[1].data = Buffer.alloc(33).toString('base64').slice(1);
 
-      const result = validatePublicKeys(publicKeys);
+        const result = validatePublicKeys(publicKeys);
 
-      expectJsonSchemaError(result);
+        expectJsonSchemaError(result);
 
-      const [error] = result.getErrors();
+        const [error] = result.getErrors();
 
-      expect(error.dataPath).to.equal('.data');
-      expect(error.keyword).to.equal('maxLength');
+        expect(error.dataPath).to.equal('.data');
+        expect(error.keyword).to.equal('minLength');
+      });
+
+      it('should be no longer than 44 character', () => {
+        publicKeys[1].data = `${Buffer.alloc(33).toString('base64')}=`;
```

Comment:
I wouldn't use padding to check the length. It's kinda ambiguous coz we do not allow paddings by pattern. So it's kinda breaks both rules here. I would suggest to remove paddings in all tests except the test against pattern.